gcp_genotype_storage/queries.sql

```diff
@@ -0,0 +1,31 @@
+SELECT DISTINCT 
```

Comment:
What does this script does and where is it used?

Reply:
It is not used. Deleted.